### PR TITLE
Update packetbeat index to [packetbeat-]YYYY.MM.DD

### PR DESCRIPTION
**Pull request by monicasarbu**
Update index pattern name in the doc.

### CODE REVIEW

**Comment by darxriggs** [comment]
Do you really want to delete `packetbeat.dev.yml` in this PR?

File: docs/gettingstarted.asciidoc [hunk 1]
```diff
@@ -309,13 +309,13 @@ it from the `config/kibana.yml` file.
 Now point your browser to port 5601 and you should see the Kibana web
 interface.
 
-The first thing you need to do is to configure the index pattern. Select *Use event times to create index names*, in
-the *Index name or pattern* field enter `[packetbeat-]YYYY.MM.DD` and for the *Time-field
+The first thing you need to do is to configure the index pattern. In
+the *Index name or pattern* field enter `packetbeat-*` and for the *Time-field
```
Comment (ruflin) [inline docs/gettingstarted.asciidoc]: @monicasarbu Do we switch back to the old pattern?

**Comment by monicasarbu** [comment]
@darxriggs I agree it's not the best time to delete the dev.yml file. I was a bit lazy to create a PR only for that. For all the beats, the configuration .dev.yml file is not used anymore. I think it has more sense to still keep it for development and every developer adjusts the .dev.yml as he/she wants. That's why is added to .gitignore.

**Comment by monicasarbu** [comment]
@ruflin thanks for noticing it. I fixed it now.

**Pull request merged**